package.json

```diff
@@ -5,12 +5,14 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "circom": "^0.5.34",
-    "circomlib": "^0.4.1",
-    "dotenv": "^8.2.0",
-    "snarkjs": "^0.3.51"
+    "hardhat": "^2.2.0",
+    "@darkforest/hardhat-circom": "../dark-forest/darkforest/packages/hardhat-circom"
```

Comment:
Can we avoid relative path here? or maybe add some instruction for prerequisite installation in README?

Reply:
Yes. its relative to an optional base path (or made absolute to the hardhat root dir)
Sorry I dont have it published yet, but this is what a fully overriden option set might look like (I was trying to keep this repo simple looking)


```js
module.exports = {
  circom: {
    verifierTemplatePath: './contracts/Verifier.sol.template',
    verifierOutName: 'Verifier.sol', // Output path is hardcoded to hardhat artifact directory
    circuitInputBasePath: './circuits/',
    circuitOutputBasePath: '../client/public/',
    circuits: [
      {
        name: 'init',
        circuit: 'init/circuit.circom',
        input: 'init/input.json',
        ptau: 'init/pot15_final.ptau',
        wasm: 'circuits/init/circuit.wasm',
        zkey: 'init.zkey',
        beacon: '0102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f',
      },
    ],
  },
};
```

Reply:
Now published as 1.0.0
https://github.com/projectsophon/hardhat-circom/releases/tag/v1.0.0